src/dmd/astcodegen.d

```diff
@@ -41,12 +41,31 @@ struct ASTCodegen
 
     alias MODFlags                  = dmd.mtype.MODFlags;
     alias Type                      = dmd.mtype.Type;
-    alias Tident                    = dmd.mtype.Tident;
```

Comment:
@ibuclaw why did you remove those aliases in #8591?

Reply:
They weren't, they were reordered alphabetically.

Reply:
Right, nevermind.